src/Umbraco.Forms.Integrations.Crm.Hubspot/HubspotComposer.cs

```diff
@@ -22,6 +23,21 @@ public void Compose(IUmbracoBuilder builder)
 
             builder.WithCollectionBuilder<WorkflowCollectionBuilder>()
                 .Add<HubspotWorkflow>();
+
+            // Generate Swagger documentation for Zapier API
+            builder.Services.Configure<SwaggerGenOptions>(options =>
+            {
+                options.SwaggerDoc(
+                    Constants.ManagementApi.ApiName,
+                    new OpenApiInfo
+                    {
+                        Title = Constants.ManagementApi.ApiTitle,
+                        Version = "Latest",
+                        Description = $"Describes the {Constants.ManagementApi.ApiTitle} available for handling Hubspot automation and configuration."
```

Comment:
Can we use `HubSpot` naming as convention whenever referencing their name publicly?